### ISSUE TITLE
Bugged task: Homecoming celebration

### ISSUE THREAD

**Issue by Orphiex** [loiste-interactive/Obenseuer-Issues#785]
The "Homecoming Celebration" task is seriously bugged in this build of the game; I think it may be unfinishable. Issues are as follows:

1. Upon taking the quest item from the "gift box" storage item on the desk, the optional task does not complete
2. Upon leaving and re-entering the Deekula C building, the gift box disappears from the desk, regardless of whether or not the quest item has been removed; leaving and re-entering the building again does not seem to resolve this issue
3. Should the quest item be registered as "stolen", it does not seem to be usable at a Chemical Lab, regardless of whether using the item from the gift box or an identical item acquired elsewhere

Screenshots available upon request.

Also, after leaving and re-entering the building, entering the quest area seems to trigger an unavoidable instance of trespassing; using the intercom no longer provides any dialogue to avoid this. Not a major issue, but possibly related to the gift box's disappearance.

**Comment by Nextej** [comment]
Taking "Mushroom extract" recipe (needed later on for the task) at Deekula A grants a seemingly random Oven recipe instead.

**Comment by Oskutin** [comment]
Fixed the recipe

**Comment by Nextej** [comment]
Granting access to free roam around the gang's base once adds additional burglary points presumably after map reload. Otherwise there's no negative reaction towards that from the henchman nor is there option to ask for the access again, seems unintentional.

**Comment by Oskutin** [comment]
Got logfile for this?

**Comment by Nextej** [comment]
Ville doesn't give you a recipe for "Poisoned Socca Junior", as the crafting requires to have the recipe to move on with the task.

**Comment by Orphiex** [comment]
> Ville doesn't give you a recipe for "Poisoned Socca Junior", as the crafting requires to have the recipe to move on with the task.

Yeah, I didn't get a recipe, but I figured that it was one of the recipes that I could unlock by putting the right ingredients in the Chemical Lab. I didn't have the Mushroom Extract recipe when I tried this, but putting the shroom bag and a suitable container in the Chemical Lab let me make the Mushroom Extract anyway.

**Comment by Nextej** [comment]
Yeah, that's Mushroom Extract, but Ville meant to give you a recipe for "Poisoned Socca Junior" which does require to have a recipe and cannot be "guessed", it take mushrooms extract, syringe and evil bears combined with normal socca junior.

**Comment by Oskutin** [comment]
Fixed ville not giving recipe

**Comment by Oskutin** [comment]
1. The objective is triggered when you leave the location with the bottle.
